[2024-05-05T15:43:04.142+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-05T15:43:04.457+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_complex.update_entry manual__2024-05-05T15:42:35.204677+00:00 [queued]>
[2024-05-05T15:43:04.517+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_complex.update_entry manual__2024-05-05T15:42:35.204677+00:00 [queued]>
[2024-05-05T15:43:04.528+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-05T15:43:04.605+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): update_entry> on 2024-05-05 15:42:35.204677+00:00
[2024-05-05T15:43:04.645+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'example_complex', 'update_entry', 'manual__2024-05-05T15:42:35.204677+00:00', '--job-id', '13', '--raw', '--subdir', '/home/***/.local/lib/python3.12/site-packages/***/example_dags/example_complex.py', '--cfg-path', '/tmp/tmp7d2ggt1q']
[2024-05-05T15:43:04.643+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=197) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-05T15:43:04.656+0000] {standard_task_runner.py:63} INFO - Started process 215 to run task
[2024-05-05T15:43:04.647+0000] {standard_task_runner.py:91} INFO - Job 13: Subtask update_entry
[2024-05-05T15:43:05.163+0000] {task_command.py:426} INFO - Running <TaskInstance: example_complex.update_entry manual__2024-05-05T15:42:35.204677+00:00 [running]> on host edcd6c8e3dad
[2024-05-05T15:43:05.839+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_complex' AIRFLOW_CTX_TASK_ID='update_entry' AIRFLOW_CTX_EXECUTION_DATE='2024-05-05T15:42:35.204677+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-05T15:42:35.204677+00:00'
[2024-05-05T15:43:05.840+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-05T15:43:05.841+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-05T15:43:05.853+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo update_entry']
[2024-05-05T15:43:05.973+0000] {subprocess.py:86} INFO - Output:
[2024-05-05T15:43:06.023+0000] {subprocess.py:93} INFO - update_entry
[2024-05-05T15:43:06.028+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-05T15:43:06.029+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-05T15:43:06.113+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=example_complex, task_id=update_entry, execution_date=20240505T154235, start_date=20240505T154304, end_date=20240505T154306
[2024-05-05T15:43:06.290+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-05T15:43:06.386+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-05T15:43:06.400+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
